frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelineCreateRuns.cy.ts

```diff
@@ -170,12 +170,14 @@ describe('Pipeline create runs', () => {
       cy.findByTestId('duplicate-name-help-text').should('be.visible');
       createRunPage.fillName('New run');
       createRunPage.fillDescription(veryLongDesc);
-      createRunPage.findExperimentSelect().should('contain.text', 'Select an experiment');
-      createRunPage.findExperimentSelect().should('not.be.disabled').click();
+      createRunPage.experimentSelect
+        .findToggleButton()
+        .should('contain.text', 'Select an experiment');
+      createRunPage.experimentSelect.findToggleButton().should('not.be.disabled').click();
       createRunPage.selectExperimentByName('Test experiment 1');
-      createRunPage.findPipelineSelect().should('not.be.disabled').click();
+      createRunPage.pipelineSelect.findToggleButton().should('not.be.disabled').click();
       createRunPage.selectPipelineByName('Test pipeline');
-      createRunPage.findPipelineVersionSelect().should('not.be.disabled');
+      createRunPage.pipelineVersionSelect.findToggleButton().should('not.be.disabled');
```

Comment:
This is a good use-case for showing how the code adjusts for the delegation of the sub-selectors in a component.